spec/vcloud/edge_gateway/edge_gateway_configuration_spec.rb

```diff
@@ -139,15 +149,17 @@ module EdgeGateway
 
       end
 
-      context "firewall config has changed and nat & load_balancer configs are absent" do
+      context "firewall and VPN config has changed and nat & load_balancer configs are absent" do
```

Comment:
After the block starting on 165, we should add an 'it' block, e.g. "proposed config contains VPN config in the form expected"